models/intermediate/shopify__inventory_level__aggregates.sql

```diff
@@ -0,0 +1,103 @@
+with order_lines as (
+
+    select *
+    from {{ var('shopify_order_line') }}
+),
+
+fulfillment as (
+
+    select *
+    from {{ var('shopify_fulfillment') }}
+),
+
+orders as (
+
+    select *
+    from {{ var('shopify_order') }}
+    where not coalesce(is_deleted, false)
+), 
+
+{% if fivetran_utils.enabled_vars(vars=["shopify__using_order_line_refund", "shopify__using_refund"]) %}
+refunds as (
+
+    select *
+    from {{ ref('shopify__orders__order_refunds') }}
+
+), refunds_aggregated as (
+    
+    select
+        order_line_id,
+        source_relation,
+        sum(quantity) as quantity,
+        sum(coalesce(subtotal, 0)) as subtotal
+
+    from refunds
+    group by 1,2
+),
+{% endif %}
+
+joined as (
+
+    select
+        order_lines.order_line_id,
+        order_lines.variant_id,
+        order_lines.source_relation,
+        coalesce(fulfillment.location_id, orders.location_id) as location_id,
```

Comment:
Will there ever be a situation where both of these are not null? Why should we choose fulfillment location over the order location?

Reply:
in reality, `fulfillment.location_id` won't be null, as order locations are actually stored there. in the data, i have seen `order.location_id` only ever be null, so this is more of a sanity check (but maybe one that is more confusing than helpful?)

Reply:
I guess what is the difference between these two location_ids? I do find this coalesce to be a bit confusing and would want this to be intentional if it is included. 

When the two ids aren't null, are they typically the same value, or do they differ?

Reply:
fair, i think it should just be `fulfillment.location_id` then! that's actually where the location is stored as far as i can tell